doc/contributing/collaborator-guide.md

```diff
@@ -168,7 +171,26 @@ dismiss the objection.
 Pull requests with outstanding objections must remain open until all
 objections are satisfied. If reaching consensus is not possible, a
 collaborator can escalate the issue to the TSC by pinging `@nodejs/tsc` and
-adding the `tsc-agenda` label to the issue.
+adding the `tsc-agenda` label to the issue. The change cannot proceed without
+either reaching consensus or a TSC decision to dismiss the objection(s). If the
+TSC does choose to dismiss any objections, a clear explanation of the reasoning
+or a link to the public vote must be given in the pull request before it lands.
+
+All Collaborator objections are considered equal. There is no greater weight given to
+objections from TSC members than from any other Collaborator.
+
+Mistakes do happen from time-to-time. If a pull request is merged with an
+unresolved objection, corrective action must be taken. What corrective
+action to take will vary on the nature of that objection. Some might be
+simple to resolve just by opening a follow-up PR that addresses it; others
+may require a full revert. When in doubt, it's likely best to propose a full
+revert, but use your judgement as other options might be available. In every
+case it is most likely the correction will be fast-tracked. There are times,
+however, when taking a slower path may be preferable, or even necessary, to
+ensure appropriate consensus or stability moving forward.
```

Comment:
```suggestion
If a pull request is merged despite an unresolved objection, fix it right away:

- Minor issue? Open a follow-up PR that addresses the concern.
- Unclear or major issue? Propose a full revert.

Most corrections can be fast-tracked. If deemed necessary, take a slower route to ensure stability and consensus.
```

I would suggest to reduce the amount of auxiliary phrases because to me as a non-English speaker it reads like an essay written for our typical English exam: I only have a few words to say but the exam tells me I must write 300 words so I will expand it as much as possible to make it that long.

Reply:
How about, 

```
Mistakes do happen from time-to-time. If a pull request is merged with an
unresolved objection, it needs to be fixed. Some issues are simple that can
be fixed with a follow-up PR that addresses the concern, but others may be
more difficult and require a full revert. Most corrections can be fast-tracked.
If deemed necessary take a slower route to ensure stability and consensus.
```